src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -246,6 +252,22 @@ public void setExecuteIgnoredModules(boolean omit) {
         executeIgnoredModules = omit;
     }
 
+    /**
+     * Sets checker threads number.
+     * @param checkerThreadsNumber The checker threads number.
+     */
+    public void setCheckerThreadsNumber(int checkerThreadsNumber) {
```

Comment:
@romani told that CLI arguments should be hidden during the gsoc (https://github.com/checkstyle/checkstyle/issues/4370#issuecomment-303486328). Shouldn't ant properties also be hidden?